lib/mockito.dart

```diff
@@ -51,4 +51,8 @@ export 'src/mock.dart'
         logInvocations,
         untilCalled,
         MissingStubError,
-        FakeUsedError;
+        FakeUsedError,
+
+        //to support non-nullable return types
```

Comment:
Nit: Space after `//` and format like a sentence. https://dart.dev/guides/language/effective-dart/documentation#do-format-comments-like-sentences